lib/good_job/active_job_extensions/interrupt_errors.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+module GoodJob
+  module ActiveJobExtensions
+    module InterruptErrors
+      extend ActiveSupport::Concern
+
+      included do
+        around_perform do |_job, block|
+          raise InterruptError, "Interrupted after starting perform at '#{CurrentThread.execution_interrupted}'" if CurrentThread.execution_interrupted.present?
+
+          block.call
+        end
```

Comment:
will this override/be overridden by any `around_perform` i have in my application_job.rb?